python/packages/autogen-ext/src/autogen_ext/agents/video_surfer/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ._video_surfer import VideoSurferAgent
+
+__all__ = ["VideoSurferAgent"]
```

Comment:
Shall we also make available the tools. These tools can also be used by other agents. Exporting them also makes it possible to reference them from VideoSurferAgent's doc string.

Reply:
Good point.